### PR TITLE
Add guideline for declaring member variables of test classes

### PR DESCRIPTION
**Pull request by dey4ss**
Guideline as discussed at the roundtable. [Click for preview](https://github.com/hyrise/hyrise/blob/dey4ss/test_variables_guideline/CONTRIBUTING.md#formatting-and-naming).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,6 +90,22 @@
       `void function(const size_t /* count */) { ... }`). If the context does not provide a good name (such as
       `/* count */`), use `/* unused */`.
 
+* In general, we declare (member) variables of the same type in an own line each ([isolated declaration](https://clang.llvm.org/extra/clang-tidy/checks/readability/isolate-declaration.html)):
```
Comment (mweisgut) [inline CONTRIBUTING.md:93]: The other guidelines are rather requests addressed to the reader:
> prefix it with `init`
> comment the parameter name out
> choose clear and concise names

So, for consistency, I would prefer formulating the new guideline in a similar way:
> Prefer declaring (member) variables of the same type in a new line each

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,6 +90,22 @@
...
+  ```c++
+  class Foo {
+    ...
+    int a;
+    int b;
+  };
+  ```
+  However, certain test classes have many member variables of the same type, e.g., `std::shared_ptr<LQPColumnExpression>` for LQP node tests. Thus, we permit violations of isolated declaration in such test cases if the number of variables is high (see below). Please use common sense to decide on which format you use.
```
Comment (mweisgut) [inline CONTRIBUTING.md:101]: > Thus, we permit violations of isolated declaration in such test cases if the number of variables is high (see below).

I would change this legislative tone. Addressing the reader, something like the following might work:
```
Declarations with more than one variables can be used in such test cases if [...]
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -90,6 +90,22 @@
...
 
+* Prefer declaring (member) variables of the same type in a new line each ([isolated declaration](https://clang.llvm.org/extra/clang-tidy/checks/readability/isolate-declaration.html)):
+  ```c++
+  class Foo {
+    ...
+    int a;
+    int b;
+  };
+  ```
+  However, certain test classes have many member variables of the same type, e.g., `std::shared_ptr<LQPColumnExpression>` for LQP node tests. Declarations with more than one variables can be used in such test classes if the number of variables is high (see below). Please use common sense to decide on which format you use.
```
Comment (mweisgut) [inline CONTRIBUTING.md:101]:
```suggestion
  However, certain test classes have many member variables of the same type, e.g., `std::shared_ptr<LQPColumnExpression>` for LQP node tests. Declarations with more than one variable can be used in such test classes if the number of variables is high (see below). Please use common sense to decide on which format you use.
```

**Comment by Bouncner** [comment]
Do we enforce separate lines via format?

**Comment by dey4ss** [comment]
> Do we enforce separate lines via format?

You mean that 
```c++
int a; int b;
```

becomes
```c++
int a;
int b;
```
?

Just tried it, `format.sh` adds the line break.

**Review by Bouncner — APPROVED** [review]
LGTM. @mweisgut : anything to add from your side?

**Review by mweisgut — APPROVED** [review]
lgtm

**Pull request merged**